frontend/src/modules/EditZing/Components/GroupGrid.tsx

```diff
@@ -60,8 +62,23 @@ export const GroupGrid = ({
     <Grid item xs={12} sm={6} md={4} lg={3}>
       <StyledGroupContainer
         ref={drop}
-        style={{ opacity: isOver ? '0.6' : '1' }}
+        style={{
+          opacity: isOver ? '0.6' : '1',
+        }}
       >
+        <Checkbox
+          sx={{
+            color: '#898992',
+            '&.Mui-checked': {
+              color: '#898992',
+            },
+            float: 'right',
+            width: '5 % ',
```

Comment:
I highly recommend you don't use "float" for styling in this case. What you want can be accomplished by using [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/), which is a much better, more modern alternative.